com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+# High Definition Range (HDR) Output in HDRP
```

Comment:
Definition > Dynamic

Reply:
@FrancescoC-unity Thoughts?

Reply:
100% correct, did the change, but HDRP sounds super close to HDR and sometimes my head gets mixed up :P